pkg/portal/cluster/machines_test.go

```diff
@@ -97,3 +111,148 @@ func TestMachines(t *testing.T) {
 		t.Fatal(r)
 	}
 }
+
+func TestVMAllocationStatus(t *testing.T) {
+	ctx := context.Background()
+	controller := gomock.NewController(t)
+	mockResourcesClient := mock_features.NewMockResourcesClient(controller)
+	mockVirtualMachinesClient := mock_compute.NewMockVirtualMachinesClient(controller)
+	NewResourceClientFunction = func(environment *azureclient.AROEnvironment,
+		subscriptionID string,
+		authorizer autorest.Authorizer) features.ResourcesClient {
+		return mockResourcesClient
+	}
+	NewVirtualMachineClientFunction = func(environment *azureclient.AROEnvironment, subscriptionID string, authorizer autorest.Authorizer) compute.VirtualMachinesClient {
+		return mockVirtualMachinesClient
+	}
+
+	defer func() {
+		NewResourceClientFunction = features.NewResourcesClient
```

Comment:
```suggestion
		NewResourceClientFunction = features.NewResourcesClient
		NewVirtualMachineClientFunction = compute.NewVirtualMachinesClient
```
Both assignments should be reversed.

Reply:
Ohh yes .. my bad .. made the changes.